cmk/base/plugins/agent_based/cisco_asa_failover.py

```diff
@@ -14,142 +15,147 @@
 # .1.3.6.1.4.1.9.9.147.1.2.1.1.1.4.6  'Active unit'
 # .1.3.6.1.4.1.9.9.147.1.2.1.1.1.4.7  'Standby unit'
 
+
+# [['Failover LAN Interface', '2', 'LAN_FO GigabitEthernet0/0.777'],
+#  ['Primary unit', '9', 'Active unit'],
+#  ['Secondary unit (this device)', '10', 'Standby unit']]
+
+# failover off/ not configured
+# [['Failover LAN Interface', '3', 'not Configured'],
+#  ['Primary unit', '3', 'Failover Off'],
+#  ['Secondary unit (this device)', '3', 'Failover Off']]
+
 from dataclasses import dataclass
-from typing import Any, Mapping, Optional
+from typing import List, Mapping, Any, Optional
+
+from cmk.base.plugins.agent_based.agent_based_api.v1.type_defs import (
+    DiscoveryResult,
+    CheckResult,
+    StringTable,
+)
 
 from cmk.base.plugins.agent_based.agent_based_api.v1 import (
-    any_of,
-    contains,
     register,
-    Result,
     Service,
+    Result,
+    State,
     SNMPTree,
     startswith,
-    State,
-)
-from cmk.base.plugins.agent_based.agent_based_api.v1.type_defs import (
-    CheckResult,
-    DiscoveryResult,
-    StringTable,
+    contains,
+    any_of,
 )
 
 
+def get_cisco_asa_state_name(st: str) -> str:
```

Comment:
Why did you modify this? Please leave it as it is (defined further below, marked as private, the dict defined outside the function and using an f-string).

Reply:
need the dict only in the litle helper function..., changed it back.

> using an f-string
from your guidlines: Use classic format strings (%s) for the time being. We'll move over to the new format() syntax in the future, but for the moment we'd like to stay consistent.